packages/grafana-ui/src/components/Table/BarGaugeCell.tsx

```diff
@@ -20,7 +21,7 @@ const defaultScale: ThresholdsConfig = {
 export const BarGaugeCell: FC<TableCellProps> = props => {
   const { field, column, tableStyles, cell, cellProps } = props;
 
-  let { config } = field;
+  let config = getFieldConfigWithMinMax(field, false);
```

Comment:
'false' will use the global min/max -- that matches the current behavior.  However, we likely want a panel option that will toggle this from the whole table vs the single column